ksql-package/pom.xml

```diff
@@ -46,6 +46,11 @@
             <groupId>io.confluent.ksql</groupId>
             <artifactId>ksql-engine</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.confluent.ksql</groupId>
+            <artifactId>ksql-tools</artifactId>
+            <version>4.1.0-SNAPSHOT</version>
```

Comment:
You should use `${project.version}` here instead and also fix clickstream-demo version above